RDC & RIA Page Changes (#1256)
Roblox/creator-docs: content/en-us/creator-programs/rdc.md

---
title: Roblox Developers Conference (RDC)
description: RDC is an invite-only, hybrid (in-person and virtual) event that brings together Roblox developers, brands, and creators worldwide.
---

RDC is an invite-only event that brings together Roblox developers, brands, and creators worldwide. It features networking opportunities, in-depth breakout sessions, product demonstrations, and office hours—all designed to empower creators and highlight the future of Roblox.

<figure>
<Chip
    color="error"
    label="Status: Closed"
    size="medium"
    variant="filled"/>
</figure><br />

For some examples of Roblox Developers Conference keynotes, talks, and Q&A sessions, see the [RDC 2025 YouTube playlist](https://youtube.com/playlist?list=PLuEQ5BB-Z1PJ8Du1rJ_JaMpW1W_G2T_lm).

## Invitations

> Eligible creators annually receive invites to the Roblox Developers Conference. These invites can be for in-person attendance or for virtual viewing. Keep an eye on the DevForum for updates regarding the conference each year. We announce "Save the Date" when registration opens.

Reviewer: ```suggestion
Eligible creators annually receive invites to the Roblox Developers Conference. These invites can be for in-person attendance or for virtual viewing. Keep an eye on the developer forum for updates regarding the conference and the "Save the Date" announcement for when registration opens.
```
